GrandCentralBoard/Widgets/Harvest/UIColor+BillingColor.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIColor+BillingColor.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 18.05.2016.
+//  Copyright © 2016 Macoscope. All rights reserved.
+//
+
+import UIKit
+
+
+extension UIColor {
+    static func hoursNormalColor() -> UIColor {
+        return UIColor(red: 35/255, green: 208/255, blue: 165/255, alpha: 1)
```

Comment:
It equals `gcb_greenColor()` from `GCBCore`, which is our theme's main color. How about using it in this place?